### PR TITLE
fix: blogpost series a announcement

### PR DESCRIPTION
**Pull request by BlueAcidFrog**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/markdown/blog/Ory-finance announcment.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+title: >
+  Ory lands $22.5M series A funding 
+
+seo:
+  description: >
+    The series A funding round was led by Insight Partners, Balderton Capital, and In-Q-Tel. Ory has now raised a total of $25 million in funding.
+
+  title: >
+    OAuth2 for Mobile Apps and Single Page Apps Best Practices (2020)
```
Comment (vinckr) [inline src/markdown/blog/Ory-finance announcment.md:12]:
```suggestion
    OAuth2 for Mobile Apps and Single Page Apps Best Practices (2020)
```

The page title needs to be updated.

File: src/markdown/blog/Ory-finance announcment.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+publishedAt: '2021-12-17'
+author: Tobias Sahl
+
+teaser: >
+  This article is a company annoucment that informs about the Series A investment.
```
Comment (vinckr) [inline src/markdown/blog/Ory-finance announcment.md:18]:
```suggestion
  This article is a company announcement that informs about the Series A investment.
```

File: src/markdown/blog/Ory-finance announcment.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+overline: Company Announcment
```
Comment (vinckr) [inline src/markdown/blog/Ory-finance announcment.md:20]:
```suggestion
overline: Ory Series A Funding
```

File: src/markdown/blog/Ory-finance announcment.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+---
+path: '/test/'
```
Comment (vinckr) [inline src/markdown/blog/Ory-finance announcment.md:2]:
```suggestion
path: '/ory-series-a-funding-update/'
```

**Review by vinckr — CHANGES_REQUESTED** [review]
Looking good already, 
some changes in the frontmatter are required :-)

**Comment by vinckr** [comment]
Oh and the file name needs to be the same as the 
`path`
and all small letters with no spaces 
so e.g.
`Ory-finance announcment.md`   -> `ory-series-a-funding-update.md`

File: src/markdown/blog/Ory-finance announcment.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+title: >
+  Ory lands $22.5M series A funding 
+
+seo:
+  description: >
+    The series A funding round was led by Insight Partners, Balderton Capital, and In-Q-Tel. Ory has now raised a total of $25 million in funding.
+
+  title: >
+    OAuth2 for Mobile Apps and Single Page Apps Best Practices (2020)
```
Comment (BlueAcidFrog) [inline src/markdown/blog/Ory-finance announcment.md:12]:
```suggestion
    Series a funding announcement for Ory
```

Comment (BlueAcidFrog) [inline src/markdown/blog/Ory-finance announcment.md]:
```suggestion
    Series A funding announcment 
```

**Review by vinckr — APPROVED** [review]
LGTM

**Pull request merged**